docs/instrumentation-list.yaml

```diff
@@ -1959,26 +1965,32 @@ internal:
     name: io.opentelemetry.opentelemetry-api-1.0
 custom:
 - name: external-annotations
+  description: |
+    The external-annotations instrumentation acts as a "shim" that automatically instruments methods annotated with custom or third-party tracing annotations. This is particularly useful if you have existing annotations (such as a custom @Trace or any third-party annotation) that you want to leverage with OpenTelemetry. At runtime, this module recognizes those annotations and applies the appropriate OpenTelemetry instrumentation logic, including span creation and context propagation. Covers many common vendor annotations by default, and additional annotations can be targeted using the configuration property: "otel.instrumentation.external-annotations.include"
   source_path: instrumentation/external-annotations
   scope:
     name: io.opentelemetry.external-annotations
 - name: opentelemetry-extension-annotations-1.0
+  description: |
+    Instruments methods annotated with Opentelemetry extension annotations, such as @WithSpan and @SpanAttribute.
```

Comment:
adding `target_versions` to some of these might be interesting, e.g. this one is a bit confusing to a casual reader what the difference is between it and `opentelemetry-extension-annotations`

Reply:
added target versions back in for all instrumentation classifications